bin/node/runtime/src/lib.rs

```diff
@@ -557,7 +557,7 @@ impl pallet_staking::Config for Runtime {
 	type MaxNominatorRewardedPerValidator = MaxNominatorRewardedPerValidator;
 	type OffendingValidatorsThreshold = OffendingValidatorsThreshold;
 	type ElectionProvider = ElectionProviderMultiPhase;
-	type GenesisElectionProvider = onchain::OnChainSequentialPhragmen<Self>;
+	type GenesisElectionProvider = onchain::UnboundedOnchainExecution<OnChainSeqPhragmen>;
```

Comment:
```suggestion
	type GenesisElectionProvider = onchain::UnboundedExecution<OnChainSeqPhragmen>;
```

sorry for the last nits, but can we rename this also to be shorter? this is already in a module called `onchain`, no need to repeat it in names.